python_files/spark_with_baseball_data.py

```diff
@@ -0,0 +1,97 @@
+# Using Spark With Baseball DB Assignment 3
+# Will McGrath
+# October 3, 2020
+
+import os
+import sys
+
+from pyspark import StorageLevel
+from pyspark.ml import Pipeline
+from pyspark.ml.feature import CountVectorizer
+from pyspark.sql import SparkSession
+from split_column_transform import SplitColumnTransform
+
+
+def main():
+    # Setup Spark and fixing time-zone issue
+    SUBMIT_ARGS = "--packages mysql:mysql-connector-java:8.0.11 pyspark-shell"
+    END_ARG = "?zeroDateTimeBehavior=CONVERT_TO_NULL&useUnicode=true& \
```

Comment:
Nice catch here as well, yep I had to troubleshoot something similar here to fix for null dates when I was setting things up.

Reply:
Yeah it was a pain setting up haha